tests/evaluation/test_generator.py

```
@@ -0,0 +1,46 @@
+"""Tests for evaluation/generator.py — synthetic test pair generation."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from unittest.mock import AsyncMock, MagicMock, patch
```

Flake8 (7.3.0)

[error] 'unittest.mock.AsyncMock' imported but unused

(F401)